src/debug_clang.cc

```diff
@@ -208,79 +206,71 @@ void Debug::Clang::start(const std::string &command, const boost::filesystem::pa
             Terminal::get().async_print(std::string(buffer, n), true);
         }
       }
-      event_mutex.unlock();
       std::this_thread::sleep_for(std::chrono::milliseconds(200));
```

Comment:
That is true. This will lead to less responsive debugging actions if I'm not mistaken.